docs/index.md

```diff
@@ -0,0 +1,51 @@
+# YamlConfiger --- AMQ configurator
+
+This tool can generate a set of configuration files mainly needed for
+AMQ Broker, but it is not limited to only generating files for one product.
+
+YamlConfiger has a user facing Command Line Tool for quick and easy command line usage.
+Furthermore, it is possible to use its API in your python code.
+
+## Documentation
```

Comment:
Will be great if you can replace this part with a link to https://rh-messaging-qe.github.io/YamlConfiger/index.html

Reply:
done in a suggestion